frontend/openshift/frontend-deploy.test.param

```diff
@@ -0,0 +1,15 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../frontend/openshift/templates/frontend/frontend-deploy.yaml
+#=========================================================
+# NAME=frontend
+# APP_NAME=pims
+# ROLE=frontend
+# IMAGE_NAMESPACE=jcxjin-tools
```

Comment:
The recommendation here is to push images to each Project rather than all of them reference the images from TOOLS. The result of only using TOOLS is that if the image is deleted it will result in failures in TEST theoretically.

Reply:
@Fosol we will have to chat when you come back from training. The above doesn't match what I've heard before but maybe there has been changes to best practices